MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly

```diff
@@ -0,0 +1,105 @@
+\version "2.18.2"
+
+\header {
+  title = "Nativity of the Theotokos"
+  subtitle = "Antiphon"
+  subsubtitle = "Verse for 3rd (Sung immediately before Apolytikion)"
+  copyright = "This arrangement is free to distribute for Liturgical use"
+  tagline = "Arranged using https://oca.org/liturgics/learning-the-tones"
+}
+
+#(set-default-paper-size "letter")
+
+% Provide an easy way to group a bunch of text together on a breve
+% http://lilypond.org/doc/v2.18/Documentation/notation/working-with-ancient-music_002d_002dscenarios-and-solutions
+recite = \once \override LyricText.self-alignment-X = #-1
+
+\defineBarLine "invisible" #'("" "" "")
+global = {
+  \time 1/1 % Not used, Time_signature_engraver is removed from layout
+  \key g \major
+  \set Timing.defaultBarType = "invisible" %% Only put bar lines where I say
+}
+
+verseOne = \lyricmode {
+  Here will I dwell, for I have cho -- sen her.
+  The Most High hath hall -- ow -- ed His tab -- er -- na -- cle.
+  Ho -- ly is Thy tem -- ple,
+  won -- der -- ful in righ -- teous -- ness.
+}
+
+soprano = \relative g' {
+  \global
+  % Verse 1
+  g4 g a b b a b c2 c4 b4 r4 \bar "|"
+  g4 a b b b b b g a2 a4 a g2 \bar "||"
```

Comment:
Combine "-low-" and "-ed" so change
g4 a b b b b b g
to
g4 a b b b b g

Reply:
Ok, i guess this is what you meant above.